assets/js/analyzer.js

```diff
@@ -26,7 +26,9 @@ if(modeEnabled('analyzation')) {
         $('#morphAnalyzerInput').keydown(function (e) {
             if(e.keyCode === ENTER_KEY_CODE && !e.shiftKey) {
                 e.preventDefault();
-                analyze();
+                if($('#morphAnalyzerInput').val().trim() !== '') {
```

Comment:
I think this might actually be cleaner if the check is performed in `analyze`. There's already an early return there. `$('#morphAnalyzerInput').val()` can also probably be assigned to a variable since it gets used later. Similar comment for the rest of these changes.